process/smartContract/builtInFunctions/esdtMetaData.go

```diff
@@ -0,0 +1,65 @@
+package builtInFunctions
+
+const lengthOfESDTMetadata = 2
+
+const (
+	METADATA_PAUSED = 1
+)
+
+// ESDTGlobalMetadata represents esdt global metadata saved on system account
+type ESDTGlobalMetadata struct {
+	Paused bool
+}
+
+// ESDTGlobalMetadataFromBytes creates a metadata object from bytes
+func ESDTGlobalMetadataFromBytes(bytes []byte) ESDTGlobalMetadata {
+	if len(bytes) != lengthOfESDTMetadata {
+		return ESDTGlobalMetadata{}
+	}
+
+	return ESDTGlobalMetadata{
+		Paused: (bytes[0] & METADATA_PAUSED) != 0,
+	}
+}
+
+// ToBytes converts the metadata to bytes
+func (metadata *ESDTGlobalMetadata) ToBytes() []byte {
+	bytes := make([]byte, lengthOfESDTMetadata)
+
+	if metadata.Paused {
+		bytes[0] |= METADATA_PAUSED
+	}
+
+	return bytes
+}
+
+const (
+	METADATA_FROZEN = 1
```

Comment:
Can we move constants up, too have them in the same place? Wold provide a better overview on the possible metadata

Reply:
done